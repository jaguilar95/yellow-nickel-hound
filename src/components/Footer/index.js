import React from "react";

function Footer() {
  return (
    <footer>
      <div className="flex-row px-1">
        <p className="mx-1">
          <a href="https://github.com/jaguilar95">GitHub</a>
        </p>
        <p className="mx-1">
          <a href="https://www.linkedin.com/in/jaguilar6115/">LinkedIn</a>
        </p>
        <p className="mx-1">
          <a href="mailto: josue.aguilar1995+bootcamp@gmail.com">Email</a>
        </p>
      </div>
    </footer>
  );
}

export default Footer;
