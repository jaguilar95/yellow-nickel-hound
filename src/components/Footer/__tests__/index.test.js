import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "..";

afterEach(cleanup);

describe("Footer component", () => {
  it("renders", () => {
    render(<Footer />);
  });

  it("footer render matched snapshot", () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });
});
