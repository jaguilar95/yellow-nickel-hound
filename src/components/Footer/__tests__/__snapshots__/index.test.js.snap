// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component footer render matched snapshot 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="flex-row px-1"
    >
      <p
        class="mx-1"
      >
        <a
          href="https://github.com/jaguilar95"
        >
          GitHub
        </a>
      </p>
      <p
        class="mx-1"
      >
        <a
          href="https://www.linkedin.com/in/jaguilar6115/"
        >
          LinkedIn
        </a>
      </p>
      <p
        class="mx-1"
      >
        <a
          href="mailto: josue.aguilar1995+bootcamp@gmail.com"
        >
          Email
        </a>
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
